<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtDLGRegEdit64Class</class>
 <widget class="QDialog" name="qtDLGRegEdit64Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>[ Nanomite ] - RegEdit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qtDLGNanomite.qrc">
    <normaloff>:/DLGNanomite/Icons/IDI_MAIN.ico</normaloff>:/DLGNanomite/Icons/IDI_MAIN.ico</iconset>
  </property>
  <widget class="QPushButton" name="pbSave">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save &amp;&amp; Exit</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="lineEdit_32">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R8</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineRSI">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineRDX">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>RSI</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_16">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>RAX</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineR12">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineRCX">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>RDX</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineRDI">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineR11">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineR8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineR13">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_28">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R11</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineRSP">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineR14">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_12">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>RSP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_26">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineRAX">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_14">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>RBX</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineRBP">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_31">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>RIP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineRIP">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_21">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R14</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>RDI</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_23">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R13</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_10">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>RCX</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_18">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R12</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineRBX">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>RBP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineR9">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_25">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R9</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineR10">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_24">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>R15</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineR15">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEFlags">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>190</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_34">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>EFlags</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbExit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>400</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="qtDLGNanomite.qrc"/>
 </resources>
 <connections/>
</ui>
